src/Column/ColumnStatus.php

```diff
@@ -164,8 +164,6 @@ public function getTemplateVariables(): array
 
 	public function setReplacement(array $replacements): Column
 	{
-		throw new DataGridColumnStatusException(
-			'Cannot set replacement for Column Status. For status texts replacement use ->setOptions($replacements)'
-		);
+		throw new DataGridColumnStatusException('Cannot set replacement for Column Status. For status texts replacement use ->setOptions($replacements)');
```

Comment:
Please revert this line - 148 characters on 1 line seams too long for me. Thanks 👍